ui_framework/doc_site/src/views/table/table_example.js

```diff
@@ -93,62 +93,62 @@ export default props => (
     </GuideSection>
 
     <GuideSection
-      title="ControlledTable"
+      title="ListingTable"
```

Comment:
Should it also show one with sorting? There's also some mention of `innerTable`, should we show that here too?

Reply:
The innerTable isn't it's own component.  We could move it out, but for now, it shouldn't get it's own section in the docs view. I will give one of the headers sort attributes.